js/components/discussions/threads-form.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+<form role="form" class="clearfix animated" v-on:submit.prevent="submit">
+    <div class="form-group">
+        <label for="title-new-discussion">{{ _('Title') }}</label>
+        <input type="text" id="title-new-discussion" v-model="title" class="form-control" required />
```

Comment:
It's useful for linking the `label[for]` in order to make the label click focusing on the field ;)